docs/utils.md

```diff
@@ -33,7 +33,7 @@ Creates a new user in the system with a username, email, and password.
 utils.createNewUser({username: "alice", email: "alice@gpii.org", password: "#1 Cloudsafe!"});
 ```
 
-## `{that}.verifyPassword(userRecord, password)`
+### `{that}.verifyPassword(userRecord, password)`
```

Comment:
This one also needs a better breakdown of the arguments, and the return value.  You can make a single example of a user record early on and refer to that in each of the functions that accept a user record.

Reply:
I think this should be matched up with the preferred format now.